scripts/meta_writer.py

```diff
@@ -110,7 +110,7 @@ def get_lite_keys(telstate, capture_block_id, stream_name):
     """
     keys = []
     for key in LITE_KEYS:
-        if key.find('?') >= 0:
+        if key.find('*_') >= 0:
```

Comment:
I would just search for '*' without the underscore. For one thing that would match the documentation.